indico/modules/designer/controllers.py

```diff
@@ -177,8 +179,21 @@ def _process_args(self):
 
     def _process(self):
         title = f'{self.template.title} (copy)'
-        new_template = DesignerTemplate(title=title, type=self.template.type, data=self.template.data,
-                                        **self.target_dict)
+        new_template = DesignerTemplate(title=title, type=self.template.type, **self.target_dict)
+
+        data = self.template.data
+        image_items = [(i, item) for i, item in enumerate(data['items']) if item['type'] == 'fixed_image']
```

Comment:
I don't think the `enumerate` is needed, there's no need to use an index. When you iterate over a list of items and mutate an item, it will change the item in the list.